scripts/ci/pre_commit/pre_commit_insert_extras.py

```diff
@@ -42,7 +42,7 @@ def get_header_and_footer(extra_type: str, file_format: str) -> tuple[str, str]:
     elif file_format == "txt":
         return f"# START {extra_type.upper()} HERE", f"# END {extra_type.upper()} HERE"
     else:
-        raise Exception(f"Bad format {format} passed. Only rst and txt are supported")
+        raise ValueError(f"Bad format {format} passed. Only rst and txt are supported")
```

Comment:
Here I think ValueError is fine @uranusjr > would you agree?

Reply:
Yeah ValueError seems file here.